src/shared_modules/indexer_connector/src/monitoring.hpp

```diff
@@ -146,15 +150,32 @@ class Monitoring final
                                         }
                                         else
                                         {
+                                            logWarn(IC_MONITORING,
+                                                    "Cluster health status: '%s'",
```

Comment:
Improve: I understand that the pre-existent code queries all the nodes of the cluster with the same query, and that all of them will return the same result because all belong to the same cluster.

But considering this, in the case a cluster isn't 'green', we'd see the same message many times. I'd add the name of the server to the message to mitigate this situation.

Reply:
done